zeus/utils/revisions.py

```diff
@@ -1,6 +1,21 @@
-from zeus.exceptions import UnknownRepositoryBackend
+from dataclasses import dataclass
+from typing import List, Tuple
+
+from zeus.exceptions import UnknownRevision
 from zeus.models import Repository, Revision
-from zeus.vcs.base import UnknownRevision
+from zeus.vcs import vcs_client
+
+
+@dataclass
+class RevisionResult:
+    sha: str
+    message: str
+    author: str
+    author_date: str
+    committer: str
+    committer_date: str
```

Comment:
Isn't this supposed to be `Optional[datetime]`? From glancing, this seems correct.